### ISSUE TITLE
Build?

### ISSUE THREAD

**Issue by lquessenberry** [yodor/keneric#1]
I have been looking everywhere for this package and your source code is the only thing on the internet left for Keneric. Do you have some advice for building this and installing it in Kubuntu?

**Comment by yodor** [comment]
Yes I made a backup from somewhere and put it here so it is not lost again :). I use it to preview STL files in dolphin.
Need some time to check about kubuntu. If you are comfortable to compile from source it is as plain as "cmake ; make ; make install". I might be able to prepare a deb package but dont have the right time now.
You can check the readme folder this is the original readme from the keneric.
